### PR TITLE
docs: add overview document

### PR DESCRIPTION
**Pull request by Joyinqin**
Signed-off-by: Joyinqin <joyinqin@pku.edu.cn>
This PR is translated from: https://github.com/chaos-mesh/website-zh/pull/9

- ~~Remove useless images~~
- Add overview document

### CODE REVIEW

**Comment by Joyinqin** [comment]
@cwen0 PTAL thx~

File: docs/overview.md [hunk 1]
```diff
@@ -1,7 +1,41 @@
...
+
+## Core strengths
+
+As the industry's leading Chaos testing platform, Chaos Mesh has the following core strengths:
+
+- Stable core capabilities: Chaos Mesh originated from the core testing platform of [TiDB](https://github.com/pingcap/tidb), and inherited a lot of TiDB's existing test experience from its initial release.
+- Fully authenticated: Chaos Mesh is used in numerous companies and organizations, such as Tencent and Meituan; It is also used in the testing systems of many well-known distributed systems, such as Apache APISIX and RabbitMQ.
+- An easy-to-use system: Chaos Mesh makes full use of automation with graphical operations and Kubernetes-based usage.
+- Cloud-native: Chaos Mesh supports Kubernetes environment with its powerful automation ability.
```
Comment (cwen0) [inline docs/overview.md:20]: s/Cloud-native/Cloud Native/g?

**Review by cwen0 — COMMENTED** [review]
Rest LGTM

**Comment by cwen0** [comment]
@Joyinqin PTAL

**Pull request merged**